packages/terra-data-grid/src/FlowsheetDataGrid.jsx

```diff
@@ -86,19 +95,42 @@ function FlowsheetDataGrid(props) {
     rowHeight,
     onCellSelect,
     onClearSelectedCells,
+    intl,
   } = props;
 
+  // Replace each non-header cell that contains no content with a dash indicating "No results".
+  const parsedRows = rows.map(
```

Comment:
Maybe we could rename to something like "generateFlowsheetRows" or "getFlowsheetRows" or something similar?

Reply:
I went with 'flowsheetRows' since it is just the array itself.